src/mrinufft/operators/interfaces/cufinufft.py

```diff
@@ -841,3 +841,9 @@ def get_lipschitz_cst(self, max_iter=10, **kwargs):
         return power_method(
             max_iter, tmp_op, norm_func=lambda x: cp.linalg.norm(x.flatten()), x=x
         )
+
+    def toggle_grad_traj(self):
+        """Toggle between the gradient trajectory and the plan for type 1 transform."""
+        if self.uses_sense:
+            self.smaps = self.smaps.conj()
```

Comment:
Turns out, if we have SENSE, while we call `fourier_op.op` with toggled plans, we still need to maintain `smaps.conj()`. Fixed that here for all the usecases